shutter/view/chart.py

```diff
@@ -2,16 +2,12 @@
 
 
 class Chart():
-
     def __init__(self, frame: Frame):
-        self.running = True
-        self.minimum = 0
-        self.maximum = 100
-        self.f = 1
+        self.line = 1
         self.x_temp_2 = 50
-        self.y_temp_2 = 0  # temp_value
+        self.y_temp_2 = 0
         self.x_light_2 = 50
-        self.y_light_2 = 0  # light_value
+        self.y_light_2 = 0
```

Comment:
Misschien andere namen geven voor:
x_temp_2
y_temp_2
x_light_2
y_light_2

Beetje onduidelijk voor mij wat het doet. Of commentaar bij zetten